src/components/global/Header/ConnectWalletButton.tsx

```diff
@@ -72,17 +82,32 @@ const WalletButton = () => {
     }
   }, [connectClick])
 
+  function addToMetaMask() {
+    web3Modal.connect().then((provider) => {
+      if (!provider.isMetaMask) {
+        alert('Please install MetaMask and try again!')
+      } else {
+        provider.request({
+          method: 'wallet_watchAsset',
+          params: {
+            type: 'ERC20',
+            options: {
+              address: TOKEN_ADDRESS[1],
+              decimals: 18,
+              symbol: 'BANK',
+              image: 'https://bankless.community/images/token-metamask.svg',
+            },
+          },
+        })
+      }
+    })
+  }
+
   const rawBalance = useTokenBalance(walletWeb3ReactContext.account) ?? 0
   const commaBalance = Number(rawBalance).toLocaleString('en')
   const whaleBalance = trimCurrencyForWhales(rawBalance)
 
   const rawClaim = useUserClaimData(walletWeb3ReactContext.account ?? '')
```

Comment:
Local testing note: You will want to change this `rawClaim` amount manually to test the "Unclaimed Tokens" notice